org/mozilla/jss/tests/JSSE_SSLClient.java

```diff
@@ -274,13 +275,13 @@ private void initSocketFactory() {
                 System.setProperty("java.protocol.handler.pkgs",
                         "com.ibm.net.ssl.www.protocol.Handler");
                 java.security.Security.addProvider((java.security.Provider)
-                Class.forName("com.ibm.jsse2.IBMJSSEProvider2").newInstance());
+                Class.forName("com.ibm.jsse2.IBMJSSEProvider2").getConstructor().newInstance());
```

Comment:
Looking at this, I'm not sure why we keep it around. I'm guessing that this test case probably doesn't work on the OpenJDK any more. It definitely isn't being run by our test harness but that's alright.

I'd probably update it to match the newer style (`main(...) throws Exception` and not catch the error) if I were picky, but like I said, not being used and likely won't be used, so I wouldn't bother.

Reply:
Agree, not worth the effort.